hooks/src/index.js

```diff
@@ -186,31 +186,29 @@ export function useContext(context) {
  */
 let afterPaint = () => {};
 
-/** @type {MessageChannel} */
-let mc;
-
+/**
+ * Pending effects consumer.
+ */
 function onPaint() {
```

Comment:
Perhaps we can give this a more descriptive name? Something like `flushAfterPaintEffects()`

Reply:
That's a fair point. That would give a bit more context on what it does. Renamed them.